src/main/java/org/terasology/Constants.java

```diff
@@ -86,6 +86,7 @@
     public static final int PEDESTAL_REGION_INDEX = 2;
     public static final int LABORATORY_SLAB_REGION = 16;
 
+    public static final String AIR_BLOCK_URI = "engine:air";
```

Comment:
If you can use a `BlockUri` instead of this String there's a constant available in `BlockManager.AIR_ID`

Also, perhaps rename this class to `MooConstants`? To avoid confusion / class name classes. Plus `MooConstants` sounds funny